includes/class-wc-subscriptions-cart.php

```diff
@@ -2355,6 +2289,7 @@ public static function get_cart_shipping_method_full_label( $label, $method ) {
 	 * method to a key that's not going to get wiped by WC's method, and then later restore it.
 	 */
 	public static function maybe_restore_chosen_shipping_method() {
+		wcs_deprecated_function( __METHOD__, 'subscriptions-core 8.1.0', 'The use of this function is no longer recommended and will be removed in a future version.' );
```

Comment:
Minor detail (not essential we change anything, though, as this is a bit of a grey area), but might this be better as:

```suggestion
		wcs_deprecated_function( __METHOD__, '7.3.0', 'The use of this function is no longer recommended and will be removed in a future version.' );
```

Here's my thinking:

- From a user perspective, the product they are using is WooCommerce Subscriptions and I *believe* we currently expect the next version to be 7.3.0.
- I realize Subscriptions Core can be used elsewhere, not just in the Subscriptions product, but that feels like more of a temporary edge case (in fact, based on some recent discussion, we decided to generally use the version number from the parent plugin for things like `@since` and `@version` tags).

Reply:
>From a user perspective, the product they are using is WooCommerce Subscriptions
Very fair point. And subscriptions-core as a standalone thing hopefully won't be around much longer.

I was somewhat blindly following the most recent example I saw in the code added here: https://github.com/Automattic/woocommerce-subscriptions-core/pull/163/commits/3c4e338be8f93acc3dead2db84dbe056acf878a9

@mattallan can you think of a reason to keep these references against subscriptions-core instead of the standalone subscriptions version?

Reply:
No, you're not missing anything :) Now that we're working towards consolidating the codebases, there's no reason to keep referencing the `subscriptions-core` version.

> And subscriptions-core as a standalone thing hopefully won't be around much longer.

This! 🎉

Reply:
Thanks, Matt and Barry!